common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -37,14 +36,6 @@ const Tab = styled(Space).attrs({
     background: ${props.theme.color('warmNeutral.300')};
   `}
 
-  ${props =>
-    props.isFocused &&
-    `
-    box-shadow: ${props.isKeyboard ? props.theme.focusBoxShadow : null};
```

Comment:
Was this replaced by a change in another component? (Or was it just unnecessary?)